index.html

```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0 minimum-scale=1.0">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap' rel='stylesheet' />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" />
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Portfolio</title>
+
+</head>
+
+<body>
+
+    <header>
+        <div class="appbar-compact">
+            <div class="toolbar">
+                <div class="logo-placeholder">
+                    <a class="navbar-brand" href="#">My Logo</a>
+                </div>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </div>
+        </div>
+
+
+        <section id="headline">
+            <div class="headline">
+                <h1 class="primary-text1">I’m Fatuma Glad to see you!</h1>
+                <p class="primary-text2">I’m a software developer! I can help you build a product , feature or website
+                    Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <div class="socialmedia">
+                    <div class="supporting-text">
+                        <p class="connect">LET’S CONNECT</p>
```

Comment:
- [ ] Please use the `a` tag instead of the `p` tag then you will use this to navigate to the contact section by using `href` attribute. :smile: